ols/src/query_helpers/question_validator.py

```diff
@@ -60,9 +60,7 @@ def validate_question(
 
         logger.info(f"{conversation_id} Validating query")
 
-        bare_llm = self.llm_loader(
-            self.provider, self.model, llm_params=self.llm_params
-        )
+        bare_llm = self.llm_loader(self.provider, self.model, self.llm_params)
```

Comment:
I'm bit confused. You've removed the `llm_params` which is an arg of `load_llm`.

Reply:
solved offline